css/styles.css

```diff
@@ -1,10 +1,120 @@
 @import url('https://fonts.googleapis.com/css2?family=Barlow+Condensed:ital,wght@0,100;0,200;0,300;0,400;0,500;1,100;1,700;1,800;1,900&family=Sacramento&display=swap');
 @import 'color.css';
 
-html{
+:root {
   font-family: 'Barlow Condensed', sans-serif;
+  background-color: var(--background);
+  font-weight: 200;
 }
 
-.theme-heading{
+a {
+  text-decoration: none;
+}
+
+.theme-heading {
   font-family: 'Sacramento', cursive;
 }
+
+.landing-footer-section {
+  padding: 2rem;
+  margin: 2rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+}
+
+#side-menu {
+  position: absolute;
+  z-index: 7;
+  max-width: 20rem;
+  height: 30rem;
+  border: 2px solid var(--primary-light);
+  box-shadow: 1px 1px 4px 4px var(--primary-light);
+  color: var(--primary);
+  background-color: var(--primary-light);
+}
+
+#side-menu:li) {
+  color: var(--primary);
+}
+
+#side-menu-filter {
+  float: right;
+  position: absolute;
+  right:0;
+  z-index: 7;
```

Comment:
use variable for z-index because it can get confusing after a while to understand stacking context